### DISCUSSION TITLE
Use Twemoji or another open source emojis

### DISCUSSION THREAD

**Discussion by RobotoSkunk** [bluesky-social/atproto#2903] [Bluesky Client App]
I would like to suggest using an emoji library in the main Bluesky client, as most of the rendered emojis are system emojis, and many of these may be outdated.

An example of this is the pink heart emoji, which is missing in some versions of Windows and Android.

**Comment by surfdude29** [comment]
This is under consideration, please see PR https://github.com/bluesky-social/social-app/pull/4862
